contracts/utils/deploy.js

```diff
@@ -275,9 +277,14 @@ function deploymentWithProposal(opts, fn) {
     } else {
       // Hardcoding gas estimate on Rinkeby since it fails for an undetermined reason...
       const gasLimit = isRinkeby ? 1000000 : null;
-      for (const proposal of proposals) {
-        const { contract, signature, args } = proposal;
-        log(`Sending goverance action ${signature} to ${address}`);
+
+      const { governorAddr } = await getNamedAccounts();
+      const sGovernor = await ethers.provider.getSigner(governorAddr);
+
+      for (const action of proposal.actions) {
```

Comment:
Okay for this not to be backward compatible?

Reply:
Yes, this is a new utility method we plan on using only for new deployments. Not used by legacy deployment scripts.